magento/pwa-studio
# [docs] Add MBI to search and update site switcher

## Description

This pull request (PR) add MBI User Guide to federated search and update the link in site switcher.

## Related Issue





Closes #ISSUE_NUMBER.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to documentation home page.
2. Search type search query
3. Hit enter
4. See that MBI User Guide results are showing 



## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
